themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml

```diff
@@ -0,0 +1,182 @@
+<?php $access = $this->driver->getOverdriveAccess();?>
+<?php if ($access->code == 'od_account_noaccess'): ?>
+  <div class="hold">
+    <div class="alert alert-warning">
+      <?= $this->translate('od_code_contentnotavail') ?>
+    </div>
+  </div>
+<?php elseif ($access->code == 'od_code_login_for_avail'): ?>
+  <p class='alert-info'><strong>
+      <?= $this->transEsc('od_code_login_for_avail') ?>
+    </strong></p>
+<?php else:
+  $od_id = $this->driver->getOverdriveID();
+  $rec_id = $this->driver->getUniqueID();
+  $hold_url = $this->url('overdrive-hold');
+  $avail = $this->driver->getOverdriveAvailability();
+  $checkedOutResult = $this->driver->isCheckedOut();
+  $checkedOut = $checkedOutResult->data;
+  if (!$checkedOut):
+    $hold = $this->driver->isHeld();
+  endif; ?>
+
+  <div class="internetDetails">
+    <div class="odbrand pull-left media-left">
+      <img
+        src="https://developerportaldev.blob.core.windows.net/media/Default/images/newLogos/OverDrive_Logo_42x42_rgb.jpg">
+    </div>
+    <h3>
+      <?= $this->transEsc('Online Access') ?>
+    </h3>
+    <div class="availability">
+      <p>
+        <?php if ($avail->data->copiesAvailable > 0): ?>
+          <span class="text-success">
+            <?= $this->icon('status-available', 'icon-link__icon') ?>
+          </span>
+        <?php else: ?>
+          <span class="text-danger">
+            <?= $this->icon('status-unavailable', 'icon-link__icon') ?>
+          </span>
+        <?php endif; ?>
+
+        <?php if (isset($avail->data->availabilityType) && $avail->data->availabilityType == 'AlwaysAvailable'): ?>
```

Comment:
Looks like this code needs to be revised to match the edits in status-full.phtml related to od_copies_available. Also, in en.ini, it looks like od_copies_available needs to have the `%%copies%%` token added to the front of it.

Reply:
It looks like this comment may have gotten missed...

Reply:
Either I misunderstood what you meant or the system just didn't pick up the change because the line number is little different:  https://github.com/bpalme/vufind/blob/VUFIND-1499%2BVUFIND-1375/themes/bootstrap3/templates/RecordTab/overdriveholdings.phtml#L48

Reply:
Strange -- when I looked at this yesterday, `od_copies_available` was still missing the `%%copies%%` token in en.ini. But I see it's there now! Sometimes GitHub lags behind, and changes don't show up right away. Perhaps that is what happened. In any case, thanks for all the progress, and I'll review this again as soon as I can (probably early next week, but maybe sooner if I'm lucky ;-) ).